designs/developer/operator-virtualization-45/index.md

```diff
@@ -0,0 +1,13 @@
+---
+parent: Developer
+version: 4.5
+---
+
+# Container-native virtualization operator
+
+When the Container-native virtualization operator is installed, the Developer Experience is enhanced to support the following:
+
+- [Visualization of Event Sources in topology](operator-virtualization-45/vm-visualization.md)
+
+## Topology side-panel
+- [Design for the Resource tab for VM based resources on topology](operator-virtualization-45/vm-sidepanel-resources.md)
```

Comment:
I think this should just be vm-sidepanel-resources.md, since it's in the same folder as index.md

Reply:
Changed